### PR TITLE
improve(affiliates): Dev Mining rewards for week 22

### LINKED ISSUE #2856: Run Dev Mining rewards week 22 between 2021/04/05 and 2021/04/12
Run Dev Mining rewards for week 23 between 2021/04/05 23:00 (1617663600000) and 2021/04/12 23:00 (1618268400000).

Contract Name | EMP Address | Payout Address
-- | -- | --
  mAMPL-USD-JUNE21 | [0xaB3Aa2768Ba6c5876B2552a6F9b70E54aa256175](https://etherscan.io/address/0xaB3Aa2768Ba6c5876B2552a6F9b70E54aa256175) | 0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9
Zelda "Summer Nuts" USDC | [0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037](https://etherscan.io/address/0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037) | 0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87
Mini Mario Cash Summer | [0x7c4090170aeADD54B1a0DbAC2C8D08719220A435](https://etherscan.io/address/0x7c4090170aeADD54B1a0DbAC2C8D08719220A435) | 0x7eb1e51351380E67F641B84545bf41eCdc10298E
BTC Basis | [0x9E929a85282fB0555C19Ed70942B952827Ca4B0B](https://etherscan.io/address/0x9E929a85282fB0555C19Ed70942B952827Ca4B0B) | 0x1Da788aBB922325E0ea2533019bf09f8a7525AC0
ETH Basis | [0x6DA66C15823cFf681DaD6963fBD325a520362958](https://etherscan.io/address/0x6DA66C15823cFf681DaD6963fBD325a520362958) | 0x1Da788aBB922325E0ea2533019bf09f8a7525AC0
pxGOLD | [0xD50fbace72352C2e15E0986b8Ad2599627B5c340](https://etherscan.io/address/0xD50fbace72352C2e15E0986b8Ad2599627B5c340) | 0x9C8F094985d12d80e930d2abBB6471EDF68d3bEB
YD-ETH-DEC21 | [0x45788a369f3083c02b942aEa02DBa25C466a773F](https://etherscan.io/address/0x45788a369f3083c02b942aEa02DBa25C466a773F) | 0xDAa953f01048253A201BA9B0bd0786575f9C2468
uForex CNYUSD  | [0x0388f65C185a7E7D857BB142185381d97a4bc747](https://etherscan.io/address/0x0388f65C185a7E7D857BB142185381d97a4bc747) | 0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c
uForex EURUSD | [0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D](https://etherscan.io/address/0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D) | 0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c
apeUSD-LINK Synthetic USD [Dec 2021] | [0x14A415Dd90B63c791C5dc544594605c8bC13Bc8D](https://etherscan.io/address/0x14A415Dd90B63c791C5dc544594605c8bC13Bc8D) | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
apeUSD-UNI Synthetic USD [Dec 2021] | [0x1066E9D2E372d01A0F57bB6f231D34Ce4CEd228e](https://etherscan.io/address/0x1066E9D2E372d01A0F57bB6f231D34Ce4CEd228e) | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
apeUSD-AAVE Synthetic USD [Dec 2021] | [0xa24Ba528Be99024f7F7C227b55cBb265ecf0C078](https://etherscan.io/address/0xa24Ba528Be99024f7F7C227b55cBb265ecf0C078) | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
apeUSD-SNX Synthetic USD [Dec 2021] | [0xd60139B287De1408f8388f5f57fC114Fb4B03328](https://etherscan.io/address/0xd60139B287De1408f8388f5f57fC114Fb4B03328) | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
apeUSD-UMA Synthetic USD [Dec 2021] | [0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f](https://etherscan.io/address/0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f) | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
uGAS-JUN21 | [0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43](https://etherscan.io/address/0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43) | 0x97990B693835da58A281636296D2Bf02787DEa17
uSTONKS-APR21 | [0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C](https://etherscan.io/address/0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C) | 0x97990B693835da58A281636296D2Bf02787DEa17
OPEN Dollar [WETH Mar 2022] | [0x964Be01cCe200e168c4ba960a764cBEBa8C01200](https://etherscan.io/address/0x964Be01cCe200e168c4ba960a764cBEBa8C01200) | 0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71
Yield Dollar UMA | [0xDB2E7F6655de37822c3020a8988351CC76caDAD5](https://etherscan.io/address/0xDB2E7F6655de37822c3020a8988351CC76caDAD5) | 0xAd2B08F878C0Be4C4162A72bbF7c0f547F93bD9f
pxGOLD Synthetic Gold Expiring 31 Mar 2022 | [0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a](https://etherscan.io/address/0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a) | 0x2CA781A4dcA1f37F52696238e70A004746068F3b
pxUSD Synthetic USD Expiring 31 Mar 2022 | [0xb40BA94747c59d076B3c189E3A031547492013da](https://etherscan.io/address/0xb40BA94747c59d076B3c189E3A031547492013da) | 0xd332f9Faf23Ba97d9f0f9EE9b3d8798C49Ff269B
DEFI_PULSE_TOTAL_TVL Synthetic Token Expiring 15 April 2021 | [0x52f83ACA94904b3590669E3525d25ec75cDFf798](https://etherscan.io/address/0x52f83ACA94904b3590669E3525d25ec75cDFf798) | 0x830c5D312D507DdB066192d34dD6441737e127C8
OPEN Dollar [OCEAN Mar 2022] | [0x312Ecf2854f73a3Ff616e3CDBC05E2Ff6A98d1f0](https://etherscan.io/address/0x312Ecf2854f73a3Ff616e3CDBC05E2Ff6A98d1f0) | 0xFc41Ad4FbAa1e6Ed9690E989121122fF6755C918
Mini Mario Winter Cash | [0x56BaBEcb3dCaC063697fE38AB745c10181c56fA6](https://etherscan.io/address/0x56BaBEcb3dCaC063697fE38AB745c10181c56fA6) | 0x7eb1e51351380E67F641B84545bf41eCdc10298E
YD-BTC-DEC21 | [0x10E018C01792705BefB7A757628C2947E38B9426](https://etherscan.io/address/0x10E018C01792705BefB7A757628C2947E38B9426) | 0xDAa953f01048253A201BA9B0bd0786575f9C2468
Zelda Winter Nuts Cash | [0xCef85b352CCD7a446d94AEeeA02dD11622289954](https://etherscan.io/address/0xCef85b352CCD7a446d94AEeeA02dD11622289954) | 0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87
YD-BTC-JUN21 | [0xd9af2d7E4cF86aAfBCf688a47Bd6b95Da9F7c838](https://etherscan.io/address/0xd9af2d7E4cF86aAfBCf688a47Bd6b95Da9F7c838) | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
YD-ETH-JUN21 | [0x0f4e2a456aAfc0068a0718E3107B88d2e8f2bfEF](https://etherscan.io/address/0x0f4e2a456aAfc0068a0718E3107B88d2e8f2bfEF) | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35

If fallback prices are needed it will be shown below:
    - [0x9E929a85282fB0555C19Ed70942B952827Ca4B0B](https://etherscan.io/address/0x9E929a85282fB0555C19Ed70942B952827Ca4B0B) = $109.15
  - [0x6DA66C15823cFf681DaD6963fBD325a520362958](https://etherscan.io/address/0x6DA66C15823cFf681DaD6963fBD325a520362958) = $109
  - [0xD50fbace72352C2e15E0986b8Ad2599627B5c340](https://etherscan.io/address/0xD50fbace72352C2e15E0986b8Ad2599627B5c340) = $1732
  - [0x0388f65C185a7E7D857BB142185381d97a4bc747](https://etherscan.io/address/0x0388f65C185a7E7D857BB142185381d97a4bc747) = $0.15
  - [0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D](https://etherscan.io/address/0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D) = $1.19
  - [0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a](https://etherscan.io/address/0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a) = $1732
  - [0x52f83ACA94904b3590669E3525d25ec75cDFf798](https://etherscan.io/address/0x52f83ACA94904b3590669E3525d25ec75cDFf798) = $52.7

### PR DESCRIPTION
**Pull request by uma-mining**
**Motivation**
#2856

**Summary**

Dev Mining results for week 22 between 2021/04/05 (1617663600000) and 2021/04/12 (1618268400000)

**Details**
In order to run create a config.json with this data:
```
{
    "startTime": 1617663600000,
    "endTime": 1618268400000,
    "totalRewards": 50000,
    "empWhitelist": [["0xaB3Aa2768Ba6c5876B2552a6F9b70E54aa256175","0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9","1.2.2"],["0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037","0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87","1.2.2"],["0x7c4090170aeADD54B1a0DbAC2C8D08719220A435","0x7eb1e51351380E67F641B84545bf41eCdc10298E","1.2.2"],["0x9E929a85282fB0555C19Ed70942B952827Ca4B0B","0x1Da788aBB922325E0ea2533019bf09f8a7525AC0","2.0.1"],["0x6DA66C15823cFf681DaD6963fBD325a520362958","0x1Da788aBB922325E0ea2533019bf09f8a7525AC0","2.0.1"],["0xD50fbace72352C2e15E0986b8Ad2599627B5c340","0x9C8F094985d12d80e930d2abBB6471EDF68d3bEB","1.2.2"],["0x45788a369f3083c02b942aEa02DBa25C466a773F","0xDAa953f01048253A201BA9B0bd0786575f9C2468","2.0.1"],["0x0388f65C185a7E7D857BB142185381d97a4bc747","0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c","2.0.1"],["0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D","0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c","2.0.1"],["0x14A415Dd90B63c791C5dc544594605c8bC13Bc8D","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],["0x1066E9D2E372d01A0F57bB6f231D34Ce4CEd228e","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],["0xa24Ba528Be99024f7F7C227b55cBb265ecf0C078","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],["0xd60139B287De1408f8388f5f57fC114Fb4B03328","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],["0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],["0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43","0x97990B693835da58A281636296D2Bf02787DEa17","2.0.1"],["0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C","0x97990B693835da58A281636296D2Bf02787DEa17","2.0.1"],["0x964Be01cCe200e168c4ba960a764cBEBa8C01200","0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71","2.0.1"],["0xDB2E7F6655de37822c3020a8988351CC76caDAD5","0xAd2B08F878C0Be4C4162A72bbF7c0f547F93bD9f","2.0.1"],["0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a","0x2CA781A4dcA1f37F52696238e70A004746068F3b","2.0.1"],["0xb40BA94747c59d076B3c189E3A031547492013da","0xd332f9Faf23Ba97d9f0f9EE9b3d8798C49Ff269B","2.0.1"],["0x52f83ACA94904b3590669E3525d25ec75cDFf798","0x830c5D312D507DdB066192d34dD6441737e127C8","2.0.1"],["0x312Ecf2854f73a3Ff616e3CDBC05E2Ff6A98d1f0","0xFc41Ad4FbAa1e6Ed9690E989121122fF6755C918","2.0.1"],["0x56BaBEcb3dCaC063697fE38AB745c10181c56fA6","0x7eb1e51351380E67F641B84545bf41eCdc10298E","2.0.1"],["0x10E018C01792705BefB7A757628C2947E38B9426","0xDAa953f01048253A201BA9B0bd0786575f9C2468","2.0.1"],["0xCef85b352CCD7a446d94AEeeA02dD11622289954","0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87","2.0.1"],["0xd9af2d7E4cF86aAfBCf688a47Bd6b95Da9F7c838","0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35","2.0.1"],["0x0f4e2a456aAfc0068a0718E3107B88d2e8f2bfEF","0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35","2.0.1"]],
    "fallbackPrices": [["0x9E929a85282fB0555C19Ed70942B952827Ca4B0B","109.15"],["0x6DA66C15823cFf681DaD6963fBD325a520362958","109"],["0xD50fbace72352C2e15E0986b8Ad2599627B5c340","1732"],["0x0388f65C185a7E7D857BB142185381d97a4bc747","0.15"],["0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D","1.19"],["0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a","1732"],["0x52f83ACA94904b3590669E3525d25ec75cDFf798","52.7"]],
}
```

Then within the affiliates package run: 
`cat config.json | node apps/DevMiningRewards --network=mainnet_mnemonic`

closes #2856

### CODE REVIEW

**Review by smb2796 — APPROVED** [review]
Rewards addresses, emp whitelist and rewards sizing looks good to me

File: packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
+{
```
Comment (chrismaree) [inline packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json:1]: this file does not look correctly linted.

Comment (smb2796) [reply packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json:1]: @evaldofelipe @daywiss can you take a look at this?

**Comment by hlambur** [comment]
Sent with https://etherscan.io/tx/0xda179a0a883f73ce4ed93f4233af4ba2a7f6042f071cc2febfcb14d985921d20

File: packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json (hunk 1 above)
Comment (evaldofelipe) [reply packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json:1]: fixed on this PR for now.

**Pull request closed without merging**